exchanges/coinbene/coinbene_wrapper.go

```diff
@@ -0,0 +1,382 @@
+		if err != nil {
+			return resp, err
+		}
+		for y := range tempData.OpenOrders {
+			tempResp.Exchange = c.Name
+			tempResp.CurrencyPair = getOrdersRequest.Currencies[x]
+			if tempData.OpenOrders[y].OrderType == buy {
+				tempResp.OrderSide = exchange.BuyOrderSide
+			}
+			if tempData.OpenOrders[y].OrderType == sell {
+				tempResp.OrderSide = exchange.SellOrderSide
+			}
+			orderTime, err := time.Parse(time.RFC3339, tempData.OpenOrders[y].OrderTime)
+			if err != nil {
+				return resp, err
+			}
+			tempResp.OrderDate = orderTime
+			tempResp.Status = tempData.OpenOrders[y].OrderStatus
+			tempResp.Price = tempData.OpenOrders[y].AvgPrice
+			tempResp.Amount = tempData.OpenOrders[y].Amount
+			tempResp.ExecutedAmount = tempData.OpenOrders[y].FilledAmount
+			tempResp.RemainingAmount = tempData.OpenOrders[y].Amount - tempData.OpenOrders[y].FilledAmount
+			tempResp.Fee = tempData.OpenOrders[y].TotalFee
+			resp = append(resp, tempResp)
+		}
+	}
+	return resp, nil
+}
+
+// GetOrderHistory retrieves account order information
+// Can Limit response to specific order status
+func (c *Coinbene) GetOrderHistory(getOrdersRequest *exchange.GetOrdersRequest) ([]exchange.OrderDetail, error) {
+	var resp []exchange.OrderDetail
+	var tempResp exchange.OrderDetail
+	var tempData ClosedOrderResponse
+	if len(getOrdersRequest.Currencies) == 0 {
+		allPairs, err := c.GetAllPairs()
+		if err != nil {
+			return resp, err
+		}
+		for a := range allPairs.Data {
+			getOrdersRequest.Currencies = append(getOrdersRequest.Currencies, currency.NewPairFromString(allPairs.Data[a].Symbol))
+		}
+	}
+	var err error
+	for x := range getOrdersRequest.Currencies {
+		tempData, err = c.FetchClosedOrders(exchange.FormatExchangeCurrency(c.Name, getOrdersRequest.Currencies[x]).String(), "")
+		if err != nil {
+			return resp, err
+		}
+		for y := range tempData.Data {
+			tempResp.Exchange = c.Name
+			tempResp.CurrencyPair = getOrdersRequest.Currencies[x]
+			if tempData.Data[y].OrderType == buy {
+				tempResp.OrderSide = exchange.BuyOrderSide
+			}
+			if tempData.Data[y].OrderType == sell {
+				tempResp.OrderSide = exchange.SellOrderSide
+			}
+			orderTime, err := time.Parse(time.RFC3339, tempData.Data[y].OrderTime)
+			if err != nil {
+				return resp, err
+			}
+			tempResp.OrderDate = orderTime
+			tempResp.Status = tempData.Data[y].OrderStatus
+			tempResp.Price = tempData.Data[y].AvgPrice
+			tempResp.Amount = tempData.Data[y].Amount
+			tempResp.ExecutedAmount = tempData.Data[y].FilledAmount
+			tempResp.RemainingAmount = tempData.Data[y].Amount - tempData.Data[y].FilledAmount
+			tempResp.Fee = tempData.Data[y].TotalFee
+			resp = append(resp, tempResp)
+		}
+	}
+	return resp, nil
+}
+
+// GetFeeByType returns an estimate of fee based on the type of transaction
+func (c *Coinbene) GetFeeByType(feeBuilder *exchange.FeeBuilder) (float64, error) {
+	return 0, common.ErrFunctionNotSupported
```

Comment:
Exchange requires hardcoded offline fees https://www.coinbene.com/fee.html#/ for calculations